module/VuFind/src/VuFind/Cover/Loader.php

```diff
@@ -438,6 +453,9 @@ protected function getIdentifiers()
         if ($this->isbn && $this->isbn->isValid()) {
             $ids['isbn'] = $this->isbn;
         }
+        if (!empty($this->isbns)) {
```

Comment:
Do we actually need a separate parameter for plural isbns vs singular isbn? Would it be better to just use the existing isbn parameter but add extra logic to handle the possibility of it being either a single value or an array? That would reduce redundancy in URLs and code.

Reply:
(I realize that we probably need separate values to get passed into the actual plugins for backward compatibility... but the rest of the "plumbing" in between could probably be adjusted to work with arrays and then pass the first value as the singular instead of keeping those things separate the whole way through).

Reply:
This simplification dose make sense to. I reduce code a bit in that way.